### PR TITLE
Bump Go to 1.19 and update Gardener dep

### PR DESCRIPTION
**Pull request by elchead**
**How to categorize this PR?**

/area security
/kind technical-debt

**What this PR does / why we need it**:
update to Go 1.19
upgrade to Gardener 1.62

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator

```

### CODE REVIEW

**Comment by gardener-robot-ci-2** [comment]
Thank you @elchead for your contribution. Before I can start building your PR, a member of the organization must set the required label(s) {'reviewed/ok-to-test'}. Once started, you can check the build status in the PR checks section below.

File: cmd/remedy-controller-azure/main.go [hunk 1]
```diff
@@ -15,18 +15,22 @@
...
 
 import (
+	"os"
+
 	"github.com/gardener/remedy-controller/cmd/remedy-controller-azure/app"
 
-	controllercmd "github.com/gardener/gardener/extensions/pkg/controller/cmd"
-	"github.com/gardener/gardener/extensions/pkg/log"
+	"github.com/gardener/gardener/pkg/logger"
+
```
Comment (kon-angelo) [inline cmd/remedy-controller-azure/main.go:23]: Can you please format these imports ?

File: go.mod [hunk 2]
```diff
@@ -1,32 +1,136 @@
...
+	k8s.io/kube-aggregator v0.25.0 // indirect
+	k8s.io/kube-openapi v0.0.0-20220803162953-67bda5d908f1 // indirect
+	k8s.io/metrics v0.25.0 // indirect
+	sigs.k8s.io/controller-runtime/tools/setup-envtest v0.0.0-20220706173534-cd0058ad295c // indirect
+	sigs.k8s.io/controller-tools v0.10.0 // indirect
+	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
+	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
 replace (
```
Comment (kon-angelo) [inline go.mod:136]: I think you can remove most of these replace directives that reference old k8s version packages with the exception of client-go (at least this was the case with the other extensions until recently)

**Review by kon-angelo — CHANGES_REQUESTED** [review]

**Pull request closed without merging**